### PR TITLE
doc: block archiver

### PR DESCRIPTION
**Pull request by alexgao001**

### CODE REVIEW

File: docs/bnb-greenfield/for-developers/data-archive/block-archiver.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+# How Block Archiver Operates
+
+The Block Archiver comprises two main components:
+
+— Block Indexer: This service continuously indexes blocks from the Blockchain and stores them in Greenfield. It ensures no block is missed and that each stored block is accurate.
+— API Server: This component handles user requests for historical block data, providing seamless access to the stored blocks.
+
+![Block Archiver](../../static/asset/block-archiver.png)
+Block Archiver Solution
```
Comment (unclezoro) [inline docs/bnb-greenfield/for-developers/data-archive/block-archiver.md:28]: This line is not necessary

File: docs/bnb-greenfield/for-developers/data-archive/block-archiver.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The indexing process ensures data integrity by running a post-verification process. This process scans all uploaded blocks,
+conducts validation checks against data already stored in Greenfield, and detects any missing data.
+
+# Who Needs to Access the Block Archiver?
+
+## Node Operators
+
+Node operators requiring full sync from the genesis block need access to historical block data via the Block-Archiver. Leveraging Greenfield's robust infrastructure, they can trust the integrity and availability of the stored data.
```
Comment (unclezoro) [inline docs/bnb-greenfield/for-developers/data-archive/block-archiver.md:37]: please hightlight that we are developing a greenfield peers that serve Block Archiver as backend

File: docs/bnb-greenfield/for-developers/data-archive/block-archiver.md (hunk 1 above)
Comment (alexgao001) [reply docs/bnb-greenfield/for-developers/data-archive/block-archiver.md]: removed

File: docs/bnb-greenfield/for-developers/data-archive/block-archiver.md (hunk 2 above)
Comment (alexgao001) [reply docs/bnb-greenfield/for-developers/data-archive/block-archiver.md]: added

**Review by unclezoro — APPROVED** [review]

**Pull request merged**